getting-started-gradle/gradle.properties

```diff
@@ -0,0 +1,6 @@
+#Gradle properties
+quarkusPluginId=io.quarkus
+quarkusPluginVersion=1.8.3.Final
+quarkusPlatformGroupId=io.quarkus
+quarkusPlatformArtifactId=quarkus-universe-bom
+quarkusPlatformVersion=1.8.3.Final
```

Comment:
Quickstarts should not use the universe bom or it's a chicken-and-eggs issue during the release.

Reply:
Yes right, I also forget that as the version.